packages/amplify-graphql-model-transformer/rds-lambda/clients/mysql/MySQLClient.ts

```diff
@@ -61,25 +61,24 @@ export abstract class MySQLClient extends DBClient {
 
   private executeList = async (request: ListRequest): Promise<any> => {
     const nextOffset = request.args?.nextToken
-      ? Number.parseInt(Buffer.from(request.args.nextToken, 'base64').toString('base64'), 10)
+      ? Number.parseInt(Buffer.from(request.args.nextToken, 'base64').toString('utf-8'), 10)
       : 0;
-    const query = (await this.getClient())(request.table).select().offset(nextOffset).limit(request.args?.limit ?? 100);
-    if (request.args?.filter) {
-      Object.keys(request.args.filter).filter((key) => request.args.hasOwnProperty(key)).forEach((key) => {
-        query.whereLike(key, request.args.filter[key]);
-      });
-      this.addSortConditions(query, request);
-    }
-    const nextToken = Buffer.from((nextOffset + request.args.limit).toString());
-    return { items: (await query.returning('*')), nextToken };
+    const limit = request.args?.limit ?? 100;
+    const query = (await this.getClient())(request.table).select().offset(nextOffset).limit(limit);
+
+    this.addSortConditions(query, request);
+    const result = await query.returning('*');
```

Comment:
Is returning supported in MySQL? I remember getting an error with mutation queries.